app/plugins/adp-backend/src/service/deliveryProjectUserRouter.ts

```diff
@@ -115,6 +117,10 @@ export function createDeliveryProjectUserRouter(
       };
 
       const addedUser = await deliveryProjectUserStore.add(addUser);
+      if (addedUser.success && addedUser.value.delivery_project_name) {
+        teamSyncronizer.syncronize(addedUser.value.delivery_project_name);
```

Comment:
I think this might be better done by adding a `syncronizeById` method and renaming this one to `syncronizeByName` rather than changing the store to pull additional data from other tables